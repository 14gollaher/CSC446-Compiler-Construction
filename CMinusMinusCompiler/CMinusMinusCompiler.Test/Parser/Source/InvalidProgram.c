foo main()
{
	foo x;
	return 1.2;
}