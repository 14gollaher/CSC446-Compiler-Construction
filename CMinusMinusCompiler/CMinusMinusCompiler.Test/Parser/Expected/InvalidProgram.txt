ERROR: Line 1 Unexpected tokens in source file, expected End-of-File Token
Completed processing InvalidProgram.c